crates/ruff_python_ast/src/int.rs

```diff
@@ -177,51 +185,33 @@ impl PartialEq<i64> for Int {
 
 impl From<u8> for Int {
     fn from(value: u8) -> Self {
-        Self::small(i64::from(value))
+        Self::small(u64::from(value))
     }
 }
 
 impl From<u16> for Int {
     fn from(value: u16) -> Self {
-        Self::small(i64::from(value))
+        Self::small(u64::from(value))
     }
 }
 
 impl From<u32> for Int {
     fn from(value: u32) -> Self {
-        Self::small(i64::from(value))
-    }
-}
-
-impl From<i8> for Int {
-    fn from(value: i8) -> Self {
-        Self::small(i64::from(value))
-    }
-}
-
-impl From<i16> for Int {
-    fn from(value: i16) -> Self {
-        Self::small(i64::from(value))
-    }
-}
-
-impl From<i32> for Int {
-    fn from(value: i32) -> Self {
-        Self::small(i64::from(value))
+        Self::small(u64::from(value))
     }
 }
```

Comment:
Any specific reason why you didn't keep the `u8`, `u16` and `u32` implementations? They would have helped to reduce the size of this PR.

Reply:
I think I did, unless I'm mistaken? I removed the `i8`, `i16`, and `i32` implementations, since it's not safe to cast from a signed to an unsigned integer.

Reply:
Removing the `i8`, `i16` etc implementations makes sense. My question is more if we should have changed them to `u8`, `u16` etc instead.

Reply:
Those already exist above, right?